### PR TITLE
GCCDEV-5057: PHP 8.3//Laravel 11 support.

### PR DESCRIPTION
**Pull request by davidxhill**

### CODE REVIEW

File: .phpactor.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+    "$schema": "/Users/dhill2/.local/share/nvim/mason/packages/phpactor/phpactor.schema.json",
```
Comment (rmarganti) [inline .phpactor.json]: you know the drill

```suggestion
```

File: composer.json [hunk 2]
```diff
@@ -13,11 +13,11 @@
         }
     ],
     "require": {
-        "php": ">=8.1",
-        "doctrine/orm": "^2.11",
-        "giadc/json-api-request": "~3.0.3",
-        "giadc/json-api-response": "~3.0.0",
-        "symfony/cache": "^6.0"
+        "php": ">=8.2",
+        "doctrine/orm": "^2.14",
```
Comment (rmarganti) [inline composer.json:17]: I wonder when/if `laravel/doctrine` is going to move on to doctrime/orm:3

**Review by rmarganti — CHANGES_REQUESTED** [review]
Looks good pending stable versions

File: composer.json (hunk 2 above)
Comment (davidxhill) [reply composer.json:17]: they market themselves as `A drop-in Doctrine ORM 2 implementation for Laravel 5+ and Lumen ` So i am going to assume never.. I am just happy it gets updated at this point.

File: .phpactor.json [hunk 3]
```diff
@@ -0,0 +1,4 @@
+{
+    "language_server_phpstan.enabled": false
+}
```
Comment (rmarganti) [inline .phpactor.json]: As mentioned elsewhere, I think we should `.gitignore` these.

File: composer.json [hunk 4]
```diff
@@ -13,11 +13,11 @@
...
+        "giadc/json-api-request": "dev-php83",
+        "giadc/json-api-response": "dev-php83",
```
Comment (rmarganti) [inline composer.json]: Request got merged. Just waiting on response.

**Review by rmarganti — CHANGES_REQUESTED** [review]
Feel free to make the changes you need to and self-approve/merge when ready.

**Pull request merged**